nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ADMINISTRATIVE_YIELD_DURATION = "nifi.administrative.yield.duration";
     public static final String PERSISTENT_STATE_DIRECTORY = "nifi.persistent.state.directory";
     public static final String BORED_YIELD_DURATION = "nifi.bored.yield.duration";
+    public static final String PROCESSOR_START_TIMEOUT = "nifi.processor.start.timeout";
```

Comment:
Very true and unfortunately this is where I would go back to the "convention-over-configuration" discussion where if we declare that MILLISECONDS is a convention (the most widely used, hence natural and intuitive) for expression of all things time related then consistency would come. 
I am just afraid that we are actually asking user to do more (think more), rather then less. On top of that it creates more possibilities of spelling errors to deal with in the event user describes it in a way we can't parse, which may come with international users who I presume may naturally type things in their own language variation ('seconde', 'segundo' etc.)

Anyway, I've changed it, so we're good here, but would like this discussion to continue at some point may be on the WIKI or dev list.